src/SFA.DAS.ApprenticePortal.Web/Services/ApprenticeService.cs

```diff
@@ -38,15 +41,15 @@ public ApprenticeService(IOuterApiClient client, NotificationAccessor notificati
 
                 var model = new HomepageModel
                 {
+                    CurrentHashedApprenticeshipId = apprenticeship == null ? null : _hashingService.HashValue(apprenticeship.Id),
                     CourseName = apprenticeship?.CourseName,
                     EmployerName = apprenticeship?.EmployerName,
                     Complete = apprenticeship?.ConfirmedOn.HasValue,
                     HasStopped = apprenticeship?.IsStopped,
                     Notification = _notifications.SignificantNotification,
                     ShowConfirmMyApprenticeshipCard = (apprenticeship is { ConfirmedOn: null }),
                 };
-                model.ShowMyApprenticeshipCard = apprenticeship is {HasBeenConfirmedAtLeastOnce: true} &&
-                                                  !model.ShowConfirmMyApprenticeshipCard;
+                model.ShowMyApprenticeshipCard = apprenticeship is {HasBeenConfirmedAtLeastOnce: true}; // && !model.ShowConfirmMyApprenticeshipCard;
```

Comment:
Is there any benefit in leaving the commented out code, or should it be removed?

Reply:
no should be out.